### DISCUSSION TITLE
Configuração dos endpoints

### DISCUSSION THREAD

**Discussion by rzMatheus** [Coaktion/client-core#15] [Ideas]
Olá pessoal, parabéns @paulo-tinoco pelo projeto!

Acabei rodando o pacote em um projeto aqui e achei bem interessante a forma que foi construído.

Gostaria de saber se é a intenção dexar os endpoints customizáveis, ou seja, onde hoje são permitidos somente search, fetch, create, update, delete, será permitido configurar o nome que desejar para o endpoint?

Valeu!! 👍🏼

**Comment by lukasMacedo06** [comment]
Fala @rzMatheus , tudo bem?

Pelo que eu havia conversado com o Tinoco, a idéia é ter essa possibilidade através da extensão da classe base onde você pode adicionar novos métodos.

No **README** há um exemplo com o método `custom`
![image](https://user-images.githubusercontent.com/14172713/214849784-a8b224c8-a5fb-4995-88c7-f27a30bce7c7.png)

**Comment by rzMatheus** [comment]
Show, entendi! A ideia é manter travado um formato de CRUD para ser enviado a classe ClientBasic e na classe que extende essa os métodos internos serão os customizados. 👍🏼
